recipes/ucc/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set cuda_version = "12" %}
+
+package:
+  name: ucc
+  version: v1.2.0
+
+source:
+  url: https://github.com/openucx/ucc/archive/refs/tags/v1.2.0.tar.gz
+  sha256: c1552797600835c0cf401b82dc89c4d27d5717f4fb805d41daca8e19f65e509d
+  patches:
+    - 0001-EC-CUDA-fix-build-with-12.2-and-older-arch-847.patch
+
+build:
+  skip: True  # [(not linux) or (cuda_compiler_version != 'None')]
+  number: 0
+  missing_dso_whitelist:
+    - '*libcuda.so*'
+  ignore_run_exports_from:
+    - cuda-nvcc
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cuda-nvcc >={{ cuda_version }}
```

Comment:
This is the correct approach. We have not yet migrated staged-recipes to CUDA 12, so new recipes should build against the provided 11.8. We build for both 11.8 and 12.x on our channel, so the recipe can be updated to build against 12.x once it has its own feedstock.

Reply:
JFYI recently we added CUDA 12 support in staged-recipes ( https://github.com/conda-forge/staged-recipes/pull/25675 ). So that hopefully is working here (appears to be running on the PR)